dracut/customization/eos-install-custom-boot-splash

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+CUSTOM_SPLASH_DIR="/eos-customization/splash"
+CUSTOM_SPLASH_CONFIG="${CUSTOM_SPLASH_DIR}/plymouthd.defaults"
+PLYMOUTH_RUNTIME_DIR="/run/plymouth"
+
+if [ ! -d ${CUSTOM_SPLASH_DIR} ] ; then
+	echo "${CUSTOM_SPLASH_DIR} not found" >&2
+	exit 1
+fi
+
+if [ ! -f ${CUSTOM_SPLASH_CONFIG} ] ; then
+	echo "${CUSTOM_SPLASH_CONFIG} not found" >&2
+	exit 1
+fi
+
+theme_name=`grep Theme ${CUSTOM_SPLASH_CONFIG}  | sed 's/Theme=\(.*\)/\1/'`
```

Comment:
I knew sed wasn't great matching across multiple lines, but I with some research I found the right spell: `sed -n 's/Theme=\(.*\)/\1/p'`